### ISSUE TITLE
Channel funding pending

### ISSUE THREAD

**Issue by excd1** [LN-Zap/zap-desktop#3709]
## Description
![zap](https://user-images.githubusercontent.com/78210221/106270386-b1f9cd80-620c-11eb-95e8-46130a3bae46.png)

It's been stuck like this for days now. Anything I can do to cancel or make it go faster? Apparently it's a common issue, maybe the default fees should be changed so this doesn't happen anymore? 

## Expected Behavior

Channel to be funded

## Actual Behavior

Channel stuck on pending

## Possible Fix

Default values on slow fees to be changed? Cancel feature? Im not a programmer.

## Steps to Reproduce

- 1.
- 2.
- 3.
- 4.

## Your Environment

- Zap version:
- Operating System and version:

**Comment by brandoncollins7** [comment]
I also have this issue, my channel has been stuck confirming for over 3 weeks. I see the transaction was sent with RBF but I don't see any functionality to up the fee? How do I get my bitcoin back?

**Comment by beinghumble** [comment]
i have the same issue. any idea on how to cancel and get the funds back.
![image](https://user-images.githubusercontent.com/80086376/110071129-82793c00-7d49-11eb-8af2-c5abc3e385f8.png)

**Comment by grubeli** [comment]
same. I guess none cares

**Comment by mr-e-** [comment]
> same. I guess none cares

Really eh... I didn't release you needed to be a programmer or you lose your money in this wallet:(

**Comment by mrfelton** [comment]
You can’t cancel a transaction after
Broadcasting it. If you broadcast it with a low fee it will take some time to confirm, that’s just how blockchains work. You money is not lost, you sent it somewhere, presumably with a low fee.

**Comment by grubeli** [comment]
No, I payed highest possible fee due to I used to need a quick transfer

**Comment by bolatovumar** [comment]
> No, I payed highest possible fee due to I used to need a quick transfer

@grubeli the fees can spike and even though the wallet was telling you it would confirm quickly at the time it's not guaranteed. I understand this is not ideal but that's how it is.

**Comment by brandoncollins7** [comment]
> You can’t cancel a transaction after
> Broadcasting it. If you broadcast it with a low fee it will take some time to confirm, that’s just how blockchains work. You money is not lost, you sent it somewhere, presumably with a low fee.

Incorrect. Our transactions have been dropped from the mempool. And now require us to be a programmer to get our money back?

**Comment by grubeli** [comment]
> > No, I payed highest possible fee due to I used to need a quick transfer
> 
> @grubeli the fees can spike and even though the wallet was telling you it would confirm quickly at the time it's not guaranteed. I understand this is not ideal but that's how it is.

Why users are not aware of such problems, why you don't mention anything like this on a front page, or at least tell us how to debug/troubleshoot the problem, or what we can do?!

**Comment by grubeli** [comment]
Since there is no tx transition ID found, can I cancel it?
if not, how can I retry the operation?

**Comment by grubeli** [comment]
DONE.
Okay. I found a way to return our funds.
if your funds are stuck like mine, I mean you send them from Zap (on-chain) wallet to Zap Lightening and is stuck on Pending.
What you need is to copy your recovery words, download new app on a different PC and recover (!) your wallet from the seed numbers you copied before. if you don't have a different PC you can use virtualization software like virtualbox (https://www.virtualbox.org) to create a virtual PC and install Zap on it.

After that, when you open your Zap on that new PC you will see all your coins on On-Chain, it means you can send it back to your original account/address or any other address

**Comment by brandoncollins7** [comment]
@grubeli Smart man, incredible that a non techie has just come up with a far more simpler solution then all these devs. Good work!

**Comment by aidenmitchell** [comment]
@mrfelton I'm having a similar issue. My channel funding payment hasn't even shown up in the mempool yet, and that's supposed to be pretty instant, no?

**Comment by bolatovumar** [comment]
> @mrfelton I'm having a similar issue. My channel funding payment hasn't even shown up in the mempool yet, and that's supposed to be pretty instant, no?

@aidenmitchell yes, it's supposed to be quick assuming the fee was not below the minimum threshold nodes on the network will accept.

**Comment by mrfelton** [comment]
Are you using the lnd node built into zap or do you have it connected to your own node?

**Comment by aidenmitchell** [comment]
> Are you using the lnd node built into zap or do you have it connected to your own node?

Built into Zap. I'm also trying to send Bitcoin with Zap and it also doesn't show up in the mempool, even with High fees selected.

**Comment by 88plug** [comment]
I am new user and also have stuck $800 now - just setup 2 channels and have pending TX now for over a week. HELP!

**Comment by iafm** [comment]
I am having the same problem, how to fix it?
